testsuite/shared/src/main/java/org/jboss/as/test/integration/common/WebInfLibClass.java

```diff
@@ -0,0 +1,4 @@
+package org.jboss.as.test.integration.common;
```

Comment:
Actually if the header is missing this is mandatory for merging, I know some other projects have relaxed that requirement but the WildFly Core and WildFly repos have mixed licensing so I believe we do still require the header to correctly identify on a file by file basis.

Reply:
Sure. I will move the license header.